### PR TITLE
[risk=no][RW-3714] Fix Data Dictionary loading bug

### PR DESCRIPTION
**Pull request by ericsong**
We were getting a duplicate upload error because it was fetching via cdr name but saving with the default cdr.

Also adds an export file for the new prod CDR.

### CODE REVIEW

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -198,6 +198,9 @@ def dev_up()
     ./gradlew loadConfig -Pconfig_key=featuredWorkspaces -Pconfig_file=config/featured_workspaces_local.json
   }
 
+  common.status "Loading Data Dictionary..."
+  common.run_inline %W{docker-compose run api-scripts ./gradlew loadDataDictionary -PappArgs=false}
```
Comment (andy7i) [inline api/libproject/devstart.rb]: It looks like we're adding data dictionary to the deploy process? Is this part of the ticket or something extra?

File: api/libproject/devstart.rb [hunk 2]
```diff
@@ -1665,6 +1668,8 @@ def deploy(cmd_name, args)
     versions_file = must_get_env_value(gcc.project, :cdr_versions_json)
     update_cdr_versions_for_project("config/#{versions_file}", op.opts.dry_run)
 
+    common.run_inline %W{gradle loadDataDictionary -PappArgs=#{op.opts.dry_run ? true : false}}
```
Comment (andy7i) [inline api/libproject/devstart.rb]: I'm not clear on the difference between `deploy()` and `dev-up` and why dev-up uses docker and deploy does not.

File: api/libproject/devstart.rb (hunk 1 above)
Comment (andy7i) [reply api/libproject/devstart.rb]: I realized that this is part of 
`Revert "Remove data dictionary tasks from dev scripts. (#2693)"` which reverts reverted code.

File: api/libproject/devstart.rb (hunk 2 above)
Comment (ericsong) [reply api/libproject/devstart.rb]: I believe it does run in docker actually. The first line of `deploy()` is `ensure_docker cmd_name, args` which will run the rest of the function in a docker environment.

File: api/libproject/devstart.rb (hunk 1 above)
Comment (ericsong) [reply api/libproject/devstart.rb]: 👍 I'm reverting the commit that removed it from our dev-up and deploy tasks. It should be there IF it wasn't buggy.

**Pull request merged**